Sources/App/entrypoint.swift

```diff
@@ -1,21 +1,29 @@
 import Vapor
 import Logging
+import NIOCore
+import NIOPosix
 
 @main
 enum Entrypoint {
     static func main() async throws {
         var env = try Environment.detect()
         try LoggingSystem.bootstrap(from: &env)
         
-        let app = Application(env)
-        defer { app.shutdown() }
+        let app = try await Application.make(env)
+
+        let executorTakeoverSuccess = NIOSingletons.unsafeTryInstallSingletonPosixEventLoopGroupAsConcurrencyGlobalExecutor()
+        if !executorTakeoverSuccess {
+            app.logger.warning("Failed to install global NIO executor, using default")
```

Comment:
Is this important enough to be worth a `warning` rather than an `info` (or possibly even just a `debug`)?

Reply:
Agreed, I would probably just do

```
app.logger.info("Running with", success ? "SwiftNIO" : "standard", "Swift Concurrency default executor")
```

or something along those lines. It's not scary or bad if this fails, you'll just have slightly higher latencies.